app/pulp/app/models/task.py

```diff
@@ -194,11 +171,10 @@ def set_failed(self, exc, einfo):
         This updates the :attr: `finished_at` attribute, sets the :attr: `state` to
         :attr: `FAILED`, and sets the :attr: `result` attribute.
```

Comment:
No space between `:attr:` and its name. This is also the case for the docstring in [`set_running`](https://github.com/pulp/pulp/pull/2789/files#diff-ae04c17153454c7da50598aecabab77bR135) (I just can't put comment there)